### PR TITLE
Add release note for 0.228

### PR DESCRIPTION
**Pull request by rongrong**
# Missing Release Notes
## Rebecca Schlussel
- [x] all checked
- https://github.com/prestodb/presto/pull/13472 Allow zoom 0 Bing Tiles
## Wenlei Xie
- [x] all checked
- https://github.com/prestodb/presto/pull/13183 Optimize PartitionedOutputOperator

# Extracted Release Notes
- #13434 James Sun: Add common scalar reads to OrcInputStream
  - Streamline ORC read path for varints and fixed width numbers.
- #13452 Andrii Rosa: Reduce excessive memory usage by ExchangeClient
  - Reduce excessive memory usage by ExchangeClient
- #13499 Wenlei Xie: Limit size of Expression/RowExpression during constant folding
  - Improve coordinator stability on parsing functions that may create large constant arrays/maps/structs (e.g., `SEQUENCE`, `REPEAT`, `ARRAY[...]`, etc) by delaying the evaluation on these functions on workers.
- #13508 James Sun: Close ColumnWriters in OrcWriter when IOException is thrown
  - Fix ORC writer rollback failure due to exception thrown during rollback.
- #13519 Andrii Rosa: Move LIMIT 0 rule to its dedicated optimization phase
  - Optimization provided for LIMIT 0 queries
- #13551 James Sun: Remove RowExpression utils from SPI
  - Removed most `RowExpression` utilities to module presto-expressions.

### CODE REVIEW

**Comment by wenleix** [comment]
@rongrong For "Optimize PartitionedOutputOperator", I believe this already goes into 0.227 so no release note is required right?

cc @yingsu00

**Comment by rschlussel** [comment]
```
General Changes
------------------
* Allow Bing Tiles at zoom level 0
```

File: presto-docs/src/main/sphinx/release/release-0.228.rst [hunk 1]
```diff
@@ -0,0 +1,18 @@
+=============
+Release 0.228
+=============
+
+General Changes
+_______________
+* Reduce excessive memory usage by ExchangeClient
+* Improve coordinator stability on parsing functions that may create large constant arrays/maps/structs (e.g., `SEQUENCE`, `REPEAT`, `ARRAY[...]`, etc) by delaying the evaluation on these functions on workers.
+* Optimization provided for LIMIT 0 queries
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/release/release-0.228.rst]: Optimize queries with ``LIMIT 0``

File: presto-docs/src/main/sphinx/release/release-0.228.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Hive Changes
+____________
+* Streamline ORC read path for varints and fixed width numbers.
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/release/release-0.228.rst]: Is this a user facing change? It sounds like an internal refactoring.

File: presto-docs/src/main/sphinx/release/release-0.228.rst [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+* Fix ORC writer rollback failure due to exception thrown during rollback.
+
+Spi Changes
+___________
+* Removed most `RowExpression` utilities to module presto-expressions.
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/release/release-0.228.rst]:
```
Move most ``RowExpression`` utilities to the ``presto-expressions`` module.
```

File: presto-docs/src/main/sphinx/release/release-0.228.rst (hunk 2 above)
Comment (rongrong) [reply presto-docs/src/main/sphinx/release/release-0.228.rst]: Should we just mention performance improvements?

Comment (rschlussel) [reply presto-docs/src/main/sphinx/release/release-0.228.rst]: oh yeah definitely. didn't even realize this was a performance improvement.

**Comment by yingsu00** [comment]
> @rongrong For "Optimize PartitionedOutputOperator", I believe this already goes into 0.227 so no release note is required right?

cc @yingsu00

That’s correct. They were already in 0.227.

**Comment by rongrong** [comment]
Anybody want to approve this?

File: presto-docs/src/main/sphinx/release/release-0.228.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+* Improve coordinator stability on parsing functions that may create large constant
+  arrays/maps/rows (e.g., ``SEQUENCE``, ``REPEAT``, ``ARRAY[...]``, etc) by delaying the
+  evaluation on these functions on workers.
+* Optimize queries with ``LIMIT 0``.
+* Allow Bing Tiles at zoom level 0.
+
+Hive Changes
+____________
+* Improve ORC read performance for variants and fixed width numbers.
+* Fix ORC writer rollback failure due to exception thrown during rollback.
```
Comment (rschlussel) [inline presto-docs/src/main/sphinx/release/release-0.228.rst]: nit: fixes come first https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes

**Pull request merged**